src/plugins/intel_gpu/src/runtime/ocl/ocl_memory.cpp

```diff
@@ -414,6 +414,10 @@ void gpu_usm::unlock(const stream& /* stream */) {
 }
 
 event::ptr gpu_usm::fill(stream& stream, unsigned char pattern) {
+    if (_bytes_count == 0) {
+        GPU_DEBUG_TRACE_DETAIL << "Skip gpu_usm::fill for 0 size tensor" << std::endl;
```

Comment:
Does that condition mean that we have allocations with 0 bytes? Or allocation size is always non-zero but memory can be reinterpreted to empty tensor? IIRC usm allocations with 0 bytes may crash on some platforms, so if we have such case somewhere in the pipeline, then we need to work-around that too.

Reply:
Memory allocation size is non zero but shape is zero size.